tools/Vis_PV.py

```diff
@@ -0,0 +1,286 @@
+
+def apply_slice(slice_Properties, var, renderView1, Filter1):
+    slice1 = Slice(Input=Filter1)
+    if slice_Properties['Slice_type'] == 'Plane':
+        slice1.SliceType = 'Plane'
+        slice1.SliceOffsetValues = [0.0]
+        slice1.SliceType.Origin= slice_Properties['Origin']
+        slice1.SliceType.Normal= slice_Properties['Normal']
+    elif slice_Properties['Slice_type'] == 'Box':
+        slice1.SliceType = 'Box'
+        slice1.SliceOffsetValues = [0.0]
+        slice1.SliceType.Position= slice_Properties['Position']
+        slice1.SliceType.Rotation= slice_Properties['Rotation']
+        slice1.SliceType.Scale= slice_Properties['Scale']
+    elif slice_Properties['Slice_type'] == 'Sphere':
+        slice1.SliceType = 'Sphere'
+        slice1.SliceOffsetValues = [0.0]
+        slice1.SliceType.Center = slice_Properties['Center_s']
+        slice1.SliceType.Radius = slice_Properties['Radius_s']
+    elif slice_Properties['Slice_type'] == 'Cylinder':
+        slice1.SliceType = 'Cylinder'
+        slice1.SliceOffsetValues = [0.0]
+        slice1.SliceType.Center = slice_Properties['Center_c']
+        slice1.SliceType.Radius = slice_Properties['Radius_c']
+        slice1.SliceType.Axis = slice_Properties['Axis']
+    #--Hide previous data display before filter--
+    Hide(Filter1, renderView1)
+    xDMFReader1_Display = Show(slice1,renderView1)
+    return renderView1, slice1, xDMFReader1_Display
+
+def apply_warp(var, xDMFReader1, renderView1):
+    '''
+    Add
+    '''
+    slice1 = Slice(Input=xDMFReader1)
+    slice1.SliceType = 'Plane'
+    slice1.SliceOffsetValues = [0.0]
+    slice1.SliceType.Origin = [3.141592653589793, 3.141592653589793, 3.141592653589793]
+    slice1.SliceType.Normal = [0.0, 0.0, 1.0]
+    #xDMFReader1_Display.ColorArrayName = ['POINTS', var]
+    warpByScalar1 = WarpByScalar(Input=slice1)
+    warpByScalar1.Scalars = ['POINTS', var]
+    Hide(xDMFReader1, renderView1)
+    xDMFReader1_Display = Show(warpByScalar1, renderView1)
+    return renderView1
+
+def save_images(renderView1, xDMFReader1):
+    '''
+    Saves single image or multiple images based on number of 
+    time steps in data. One image per time step saved
+    '''
+    time_steps = xDMFReader1.TimestepValues
+    renderView1.ViewSize = [1330, 670]
+    try:
+        n_steps = len(time_steps)
+    except:
+        n_steps = 1
+        time_steps = [time_steps]
+    if n_steps ==1:
+        SaveScreenshot('image.png', renderView1)
+    elif n_steps > 1:
+        anim = GetAnimationScene()
+        anim.PlayMode = 'Snap To TimeSteps'
+        for n in range(n_steps):
+            print('loop',n)
+            anim.AnimationTime = time_steps[n]
+            view = GetRenderView()
+            SaveScreenshot('image_%s.png' %n, view)
+    return None
+
+def main():
+    '''
+    Render image(s) based on parameters specified 
+    '''
+    # Get data from file, set default camera properties
+    Input_data = load_InputData()
```

Comment:
`input_file = load_input_file`, or you could create an `InputFile` class whose constructor takes the name of the input file as an argument and then has attributes for the different input file options.